### ISSUE TITLE
[Support]: Working Offline

### ISSUE THREAD

**Issue by armindarvish** [progfolio/elpaca#209]
### Confirmation

- [X] I have read the documentation and need clarification.
- [X] I have searched previous issues to see if my question is a duplicate.
- [ ] I am  checking these without reading them.
- [X] I have checked the Elpaca github wiki to see if my issue is covered.

### Elpaca Version

Elpaca 644a30a HEAD -> master, origin/master, origin/HEAD
installer:      0.6
emacs-version:  GNU Emacs 30.0.50 (build 1, aarch64-apple-darwin22.6.0, NS appkit-2299.70 Version 13.5.2 (Build 22G91))
 of 2023-09-12
git --version:  git version 2.42.0

### Operating System

MacOS

### Description

Hi I just started testing our Elpaca, and one thing I cannot figure out is how to get it to work when i am offline. Basically packages that are in my init file and have been cloned previously are giving errors. With Straight.el, there is a prompt asking if it should skip repositories, etc. is there something similar for Elpaca. How do I skip everything if the packages are all cloned already?

**Comment by progfolio** [comment]
> Hi I just started testing our Elpaca

Hi. Thanks for giving Elpaca a shot and taking the time to fill out a support ticket.

> I cannot figure out is how to get it to work when i am offline.
> Basically packages that are in my init file and have been cloned previously are giving errors.

Could you be more specific about what you are trying to do which causes the errors?
Please also include the output of `M-x elpaca-log`.

> With Straight.el, there is a prompt asking if it should skip repositories, etc. is there something similar
> for Elpaca. How do I skip everything if the packages are all cloned already?

I'm not sure what you mean by "skip everything".
What are you trying to do which requires skipping repositories?

**Comment by armindarvish** [comment]
Actually I cannot reproduce the issue. I am guessing the repositories were not cloned for some reason.

Basically I was porting my init file from straight.el to elpaca and I got a bunch of packages installed and built. So some simple init file with a bunch of use-package declarations:

```emacs-lisp
(use-package doom-themes)
(use-pakcage evil)
```

Then I closed Emacs and a few hours later, I opened Emacs while I was offline and instead of building from the local copies, it just threw a bunch of errors at me and I thought that was because it was trying to reach the network. With straight.el, if it cannot reach the network to get the gnu-elpa recipes, etc. it asks if it should skip the repository, but here with elpaca I was just seeing failed for packages and it was not building form local copies. 

But I cannot reproduce the issue now. It works as expected. If it happens again, I'll document it for you and will reopen this issue.
